### PR TITLE
docs: update CONTRIBUTING.md

### LINKED ISSUE #1386: [DOCS] CONTRIBUTING.md needs improvement
### Description

It will be great if it includes more info about what `git add public` is and how we can include different files which we just changed for newbies like me

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by PritamSarbajna**
## Fixes Issue
Closes #1386

## Changes proposed

-  It is now more clear that why we use `git add public` as I have also mentioned that we can also use specific name of the folder we just changed
-  Added commit point separately 
-  made `Instead, stage your changes for each file/folder` line as note and also added `By using public path it means it will add all files and folders under that folder, it is better to be specific` to it

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### *It is looking like this after updating*

Screenshot (115)

## Note to reviewers

> I am facing these problems so I thought it'll be helpful who are just starting out

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1387"></a>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-
-
 ---
```
Comment (emmalearnscode) [inline CONTRIBUTING.md:1]: Removing the first two lines of the document messes up the formatting of the first section.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Just a couple of small changes, then this will be good to go.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,33 +77,46 @@ git merge upstream/main
...
 
-**8.** Stage your changes and commit:
+**8.** Stage your changes:
 
 ⚠️ **Make sure** not to commit `package.json` or `package-lock.json` file
 
-⚠️ **Make sure** not to run the commands `git add .` or `git add *`. Instead, stage your changes for each file/folder
+⚠️ **Make sure** not to run the commands `git add .` or `git add *`
+
+> Instead, stage your changes for each file/folder
```
[the remark this answers is not in the document]
Comment (PritamSarbajna) [reply CONTRIBUTING.md:86]: Please let me know if this is better
Screenshot (122)

[the remark this answers is not in the document]
Comment (emmalearnscode) [reply CONTRIBUTING.md:87]: Yes, this looks better.

[the remark this answers is not in the document]
Comment (PritamSarbajna) [reply CONTRIBUTING.md:87]: can you merge it

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**